doc/manual/types.rst

```diff
@@ -1237,24 +1237,15 @@ If you apply :func:`supertype` to other type objects (or non-type objects), a
     julia> supertype(Union{Float64,Int64})
     ERROR: `supertype` has no method matching supertype(::Type{Union{Float64,Int64}})
 
+.. _man-val-trick:
+
 "Value types"
 -------------
 
 As one application of these ideas, Julia includes a parametric type,
```

Comment:
Never understood what kind of "these ideas" are being referred to. I guess many people who just followed the `_man-val-trick` link have the same experience.
Also, I find it a little bit confusing that `Val{T}` is introduced as some sort of Julia type system quirk, whereas in fact it's just an empty immutable, and the same effect could be achieved with any user-defined type. Maybe it's worth clarifying.